docs/basics.md

```diff
@@ -170,6 +170,16 @@ other things yourself, if needed.
 
 [Read more about customizing `urql` in the "Extending & Experimenting" section.](https://formidable.com/open-source/urql/docs/extending-&-experimenting)
 
+## Suspense
+
+React has this neat feature called [`suspense`](https://reactjs.org/docs/code-splitting.html) which
+these days is used for `code-splitting` but it can be used for more. In our case when
+your query is fetching we can also utilise this `Suspense` boundary to transition
+into a loading state instead of you having to do this manually for every component.
+
+Enabling suspense can be done by going to your client and passing it the option `suspense: true`.
```

Comment:
This probably won’t just work like people expect it to unless they use the suspenseExchange 🤔